src/cli.yml

```diff
@@ -0,0 +1,15 @@
+name: wiki
+version: "0.1"
+author: Rust Leipzig
+about: Generates static HTML files based on markdown files.
+args:
+    - output-directory:
+        short: o
+        long: output-directory
+        value_name: PATH
+        help: The directory where the HTML output is generated.
+        takes_value: true
```

Comment:
This is where we have to discuss in #8 . In my view the combination of fixed folders (/input and /output) and commandline arguments is the best. Lets discuss it next thursday.